src/main/kotlin/com/arabyte/arabyteapi/domain/mypage/controller/MyPageController.kt

```diff
@@ -0,0 +1,65 @@
+package com.arabyte.arabyteapi.domain.mypage.controller
+
+import com.arabyte.arabyteapi.domain.article.dto.ArticlePreviewResponse
+import com.arabyte.arabyteapi.domain.mypage.dto.MyPageResponse
+import com.arabyte.arabyteapi.domain.mypage.dto.UpdateBasicInfoResponse
+import com.arabyte.arabyteapi.domain.mypage.dto.UpdateSubInfoResponse
+import com.arabyte.arabyteapi.domain.mypage.service.MyPageService
+import com.arabyte.arabyteapi.domain.user.entity.User
+import com.arabyte.arabyteapi.global.annotation.RequestUser
+import io.swagger.v3.oas.annotations.Operation
+import org.springframework.data.domain.Page
+import org.springframework.web.bind.annotation.*
+
+@RestController
+@RequestMapping("/mypage")
+class MyPageController(
+    private val myPageService: MyPageService,
+) {
+    @Operation(
+        summary = "마이페이지 게시물 조회",
+        description = "/articles/my로 요청하면 내가 쓴 게시물, /article/like로 요청하면 내가 좋아요를 누른 게시물을 반환합니다."
+    )
+    @GetMapping("/articles/{type}")
+    fun getMyArticles(
+        @PathVariable type: String,
+        @RequestUser user: User,
+        @RequestParam page: Int,
+        @RequestParam size: Int
+    ): Page<ArticlePreviewResponse> {
+        return myPageService.getMyPageArticles(type, user, page, size)
+    }
+
+    @Operation(summary = "닉네임 수정", description = "마이페이지에서 닉네임을 수정합니다.")
+    @PatchMapping("/nickname/update")
```

Comment:
```suggestion
    @PatchMapping("/nickname")
```

rest에서는 url에 action이 들어가면 안됩니다. action은 http method로 표현해주세요

Reply:
반영하겠습니다~